.github/workflows/_build.yaml

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) 2022 - 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
> Could this workflow be named `build.yaml` instead? Because it is a reusing-workflow, I don't think we should name it in a similar manner as a private workflow thinking

This is a private workflow, which is also a reusable workflow. So, keeping `_build.yaml` is preferred.